src/views/CardStack.js

```diff
@@ -207,11 +224,16 @@ class CardStack extends React.Component<DefaultProps, Props, void> {
           if (header && header.left) {
             return header.left;
           }
-          const { renderLeftComponent } = this.props.headerComponent.defaultProps || {};
-          if (typeof renderLeftComponent === 'function') {
-            return renderLeftComponent(props);
+          if (props.scene.index === 0 || !props.onNavigateBack) {
+            return null;
           }
-          return null;
+          const prevSceneTitle = this._getHeaderTitleForScene(props.scenes[props.index - 1]);
+          return (
+            <HeaderBackButton
```

Comment:
@grabbou but shouldn't any custom header component be able to render the back button with title? otherwise if I want to use custom header button I can't show the title, and if I want to show the title I can't use a custom back button. Probably better to just add `title` to scene.